maxim/scribe.py

```
@@ -15,49 +15,112 @@ class Scribe:
     """
 
     def __init__(self, name):
+        """Initialize a scribe logger.
+
+        Args:
+            name: The name of the logger.
+        """
         self.name = name
         self.disable_internal_logs = True
         self.logger = logging.getLogger(name)        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)